src/commonMain/kotlin/software/momento/kotlin/sdk/responses/cache/GetResponse.kt

```diff
@@ -1,31 +1,63 @@
 package software.momento.kotlin.sdk.responses.cache
 
 import software.momento.kotlin.sdk.exceptions.SdkException
-import java.nio.charset.StandardCharsets
-import java.util.Base64
+import software.momento.kotlin.sdk.internal.utils.encodeBase64
+import software.momento.kotlin.sdk.internal.utils.truncate
 
-// Sealed interface for the GetResponse
+/**
+ * Response for a cache get operation
+ */
 public sealed interface GetResponse {
 
-    // Represents a successful get operation with a value
-    public class Hit(private val value: ByteArray) : GetResponse {
-        // Convert value to a string
-        private fun valueString(): String = value.toString(StandardCharsets.UTF_8)
+    /**
+     * A successful get operation for a key that has a value.
+     *
+     * @property valueByteArray The retrieved value as a byte array.
+     */
+    public data class Hit(public val valueByteArray: ByteArray) : GetResponse {
```

Comment:
I have the `value` and `valueByteArray` fields as properties here instead of methods because that felt more in the Kotlin style of hidden getters.